src/test/java/PlaylistTest.java

```diff
@@ -0,0 +1,107 @@
+import Pages.HomePage;
+import Pages.LoginPage;
+import Pages.PlayControlsPage;
+import Pages.PlayListPage;
+import org.testng.Assert;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+public class PlaylistTest extends BaseTest {
+        @Test(priority = 1, description = "Validate new empty playlist is created")
+        @Parameters({"Playlist"})
+        public void createPlaylist (String Playlist) throws InterruptedException {
+            LoginPage loginPage = new LoginPage(driver);
+            PlayListPage playListPage = new PlayListPage(driver);
+            loginPage.login();
+
+            String notificationText = "Created playlist \"" + Playlist + ".\"";
+            playListPage.createNewPlaylist(Playlist);
+            playListPage.newPlaylistNotification();
+            Assert.assertTrue(playListPage.newPlaylistIsDisplayed());
+            Assert.assertTrue(playListPage.newPlaylistNotification().contains(notificationText));
+        }
+    @Test(priority = 2, description = "Validate song is added to playlist")
+    @Parameters({"Playlist"})
+    public void addSongToPlaylist(String Playlist) throws InterruptedException {
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+        loginPage.login();
+
+        homePage.searchSong ("Metre");
+        Thread.sleep(2000);
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.